package.json

```diff
@@ -23,6 +23,11 @@
   "homepage": "https://github.com/lumc03/workshop-api-testing-js#readme",
   "devDependencies": {
     "chai": "^4.3.6",
+    "http-status-codes": "^2.2.0",
     "mocha": "^10.0.0"
+  },
+  "dependencies": {
+    "axios": "^0.27.2",
+    "mochawesome": "^7.1.3"
```

Comment:
mochawesome should be a dev dependency, to fix that, run: `npm i -D mochawesome`